### ISSUE TITLE
Not displaying

### ISSUE THREAD

**Issue by vwbusdriver** [kapsolas/MMM-Instagram#4]
I received my access token through instagram and hooked it into the config.js file but nothing displays. the console says 'Starting node_helper for module [MMM-Instagram]'. I even tried putting the access token directly in the Instagram.js file and still nothing. Any help would be much appreciated.

**Comment by Z3R0SUMx** [comment]
I'm having this issue too

**Comment by kapsolas** [comment]
Have you guys resolved your issue with the images not displaying?

**Comment by Z3R0SUMx** [comment]
Yes! Thanks! Is there a way to make the feed random rather than in order?

**Comment by janhenrik** [comment]
@Z3R0SUMx How did you fix your problem?

**Comment by inderpartap** [comment]
Even I am having the same issue. Please help

**Comment by thiemt** [comment]
Having the same problem here. @Z3R0SUMx please tell us how you fixed it.

**Comment by Z3R0SUMx** [comment]
Tbh I don't know. It was a long time ago and my first coding project. I was trying random things and finally got it to work - sorry friend

**Comment by thiemt** [comment]
My problem was, that the facial recogtnion module was hidding it for some reason fixed it by unhiding it using the MMM-remote Module.
